datumaro/plugins/icdar_format/converter.py

```diff
@@ -167,7 +169,8 @@ def apply(self):
                 for task, task_conv in task_converters.items():
                     if item.has_image and self._save_images:
                         self._save_image(item, osp.join(
-                            self._save_dir, subset_name, IcdarPath.IMAGES_DIR,
+                            self._save_dir, IcdarPath.TASK_DIR[task],
```

Comment:
Does this mean an image can be saved multiple times, if it contains annotations from few tasks? Maybe we don't need common converter then?